dandi/bids_validator_xs.py

```diff
@@ -644,6 +650,11 @@ def validate_bids(
         regex_schema,
         debug=debug,
     )
-    write_report(validation_result)
+
+    if report_path:
+        if isinstance(report_path, str):
+            write_report(validation_result, report_path=report_path)
+        else:
+            write_report(validation_result)
```

Comment:
that is why codecov complains since this line is not covered AFAIK.
FWIW code logic is a bit confusing due to support of the `bool` value here. I would have not bothered with "default location" (via `bool`). But if it is supported -- add a test which would test this code path to make codecov happy.

Reply:
The problem is that if the logs as per the default path are to be dropped in the current working directory, there's no way to not drop them in the current directory :o)

I think it's an important feature since typing in the datetime pattern is annoying, and it makes sense to be able to rapidly generate unique logs when running the command in batch.

My current approach was to move the default path from the current working directory to `/var/tmp/` — let's see how well that goes down with windows tests.

Reply:
well, the tests pass on windows, which is cool, I guess.